nav_normal.php

```diff
@@ -0,0 +1,13 @@
+<!--nav for User page -->
```

Comment:
21/9: I also create a normal nav page so when I called it up the nav for normal user can be function. Also the link has been check so it all link to right location.